### ISSUE TITLE
problem Installation HAxm

### ISSUE THREAD

**Issue by cedric8528** [intel/haxm#477]
Hello,

I'have tried all solutions of this thread to find how to install HAXM but I have the same problem...Installation failed (in Android Studio and also via the exe of Intel). 
I have also tried "silent_install.bat....

Configuration : Windows 7 SP1

![Captur12](https://user-images.githubusercontent.com/114312429/200041982-16ff36bb-02a4-46b1-81b3-9990c57afc92.PNG)

![Capture13](https://user-images.githubusercontent.com/114312429/200042053-9e10282a-de63-4e42-8e40-c1d987dc1a4b.PNG)

![Capture14](https://user-images.githubusercontent.com/114312429/200042150-ec3ecdb1-5472-4613-a429-50b2db10df7f.PNG)

![Capture11](https://user-images.githubusercontent.com/114312429/200042181-1973acc7-82d2-46eb-a265-9e74962f2f8f.PNG)

If someone have an idea to help me :-)

Thanks

Cédric.

**Comment by cedric8528** [comment]
For People who have the same problem, I have installed an old version of HAXM...
7.5.1 works for me 

https://github.com/intel/haxm/releases/

**Comment by sskras** [comment]
What is the earliest HAXM version known to fail during the installation?
This could be useful for futher regression testing.

**Comment by sskras** [comment]
Hey @cedric8528, did you solve this?

**Comment by cedric8528** [comment]
Hello sskas,

Yes I did

All versions above 7.5.4 don't work for me... (installation failed)

7.5.4 and below works perfectly.

**Comment by wayne-ma** [comment]
Yes, indeed. For the old CPU family and Windows 7 version, please use the v7.5.1 HAXM release.
